kpops/api/registry.py

```diff
@@ -45,32 +52,52 @@ def __getitem__(self, component_type: str) -> type[PipelineComponent]:
             msg = f"Could not find a component of type {component_type}"
             raise ClassNotFoundError(msg) from ke
 
+    @staticmethod
+    def iter_component_modules() -> Iterator[ModuleType]:
+        import kpops.components
+
+        yield kpops.components
+        yield from _iter_namespace(kpops.components)
 
-def find_class(module_name: str, baseclass: type[T]) -> type[T]:
+
+def find_class(modules: Iterable[ModuleType], base: type[T]) -> type[T]:
     try:
-        return next(_find_classes(module_name, baseclass))
+        return next(_find_classes(modules, base=base))
     except StopIteration as e:
         raise ClassNotFoundError from e
 
 
-def _find_classes(module_name: str, baseclass: type[T]) -> Iterator[type[T]]:
+def import_module(module_name: str) -> ModuleType:
     module = importlib.import_module(module_name)
+    # TODO: remove? unnecessary now
```

Comment:
What about this TODO?

Reply:
I've changed the logging a bit to print out the module and filepath